packages/advi/advi.2.0.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "Active DVI Dune package!"
+description: """
+
+Active-DVI is a DVI previewer and presenter written in Objective Caml
+(http://caml.inria.fr/ocaml/) with some eye candy effects for
+presentation, support for interactive demonstrations, and embedding of
+arbitrary applications within the presentation.
+
+Active-DVI is very well suited for use in combination with WhizzyTeX
+(http://cristal.inria.fr/whizzytex/)
+"""
+maintainer: ["Didier Rémy <Didier.Remy@inria.fr>"]
+authors: [
+  "Alexandre Miquel"
+  "Jun Furuse"
+  "Didier Rémy"
+  "Pierre Weis"
+  "Xavier Leroy"
+  "Roberto Di Cosmo"
+  "Didier Le Botlan"
+  "Alan Schmitt"
+]
+license: "<name>"
+homepage: "http://advi.inria.fr"
+doc: "https://gitlab.inria.fr/remy/advi"
```

Comment:
The inria gitlab page does not seem reachable. 
```suggestion
doc: "http://advi.inria.fr/"
```

do you also have an accessible dev repository?